### PR TITLE
Feature/extractscore

### PR DESCRIPTION
**Pull request by yuna-iwata**
- I have managed to extract the score from the phaser game on button click ( there is a bug where extracting the score rerenders another game which I will work on in the next branch)
- The game is also finished to an MVP - basically working exactly like the chrome dino game - added sounds, game over, score and bird obstacle

### CODE REVIEW

**Review by louisglan — CHANGES_REQUESTED** [review]
Didn't pass netlify checks

**Review by MariamM110 — APPROVED** [review]
Great progress!

File: package.json [hunk 1]
```diff
@@ -1,27 +1,30 @@
 {
-  "name": "dino-game-frontend",
```
Comment (louisglan) [inline package.json:2]: Might be better to have this name

File: src/Pages/GamePage.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { useState } from "react";
+import HelloWorldScene from "../scenes/HelloWorldScene";
```
Comment (louisglan) [inline src/Pages/GamePage.tsx]: Could change the name from Hello World Scene but not essential

File: src/index.tsx [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
 import "./index.css";
 import App from "./App";
 import reportWebVitals from "./reportWebVitals";
-import "bootstrap/dist/css/bootstrap.css";
 
-const root = ReactDOM.createRoot(document.getElementById("root"));
-root.render(<App />);
+ReactDOM.render(
+  // <React.StrictMode>
+  //   <App />
```
Comment (louisglan) [inline src/index.tsx:9]: Good to have strictmode whilst coding because it's good for debugging

**Review by louisglan — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (yuna-iwata) [reply package.json:2]: good point thanks

**Review by louisglan — APPROVED** [review]
Looks great!

**Pull request merged**